src/worldofzuul/Player.java

```diff
@@ -5,11 +5,47 @@
  */
 package worldofzuul;
 
+import java.util.ArrayList;
+
 /**
  *
  * @author Kasper
  */
 public class Player extends Character{
+    /**
+     * The arraylist stores the items, that the player picks up. 
+     */
+    private ArrayList<Item> items = new ArrayList<Item>();
+    
+    /**
+     * The method adds items to the arraylist.
+     * @param items the item which is added to the inventory.
+     */
+    public void addItems(ArrayList items){
+        
+    }
+    /**
+     * The method displays the items currently held.
+     */
+    public void getItems() {
+        for (String item : items) {
+            System.out.println(item);
+        }    
+    }
     
+    /**
+     * The method checks if the inventory is full.
+     * @return returns a "true" or "false" value depending on how many items held.
+     */
+    private boolean isInventoryFull() {
+        
+        if (items.length() > 1) {
+            
+            return true;
+        }
+        else {
+            return false;
+        }
+    }
```

Comment:
When you have a method with a simple if statement and it just returns a boolean, the if evaluation statement could just be returned instead.
```java
private boolean isInventoryFull() {
     return (items.length() > 1);
}
```

Reply:
Fixed in 66ca6fefda976000a256c536838a2fca524eaf2d